tester_app/app/controllers/hello_world_controller.rb

```diff
@@ -0,0 +1,2 @@
+class HelloWorldController < ApplicationController
```

Comment:
This should have a controller method that sets the @hello_world_props that used for the initial render.